specification/cdn/resource-manager/Microsoft.Cdn/stable/2017-10-12/cdn.json

```diff
@@ -2249,18 +2249,18 @@
           "description": "The order in which the rules are applied for the endpoint. Possible values {0,1,2,3,………}. A rule with a lesser order will be applied before a rule with a greater order. Rule with order 0 is a special rule. It does not require any condition and actions listed in it will always be applied.",
           "type": "integer"
         },
-        "actions": {
-          "description": "A list of actions that are executed when all the conditions of a rule are satisfied.",
+        "conditions": {
+          "description": "A list of conditions that must be matched for the actions to be executed",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/DeliveryRuleAction"
+            "$ref": "#/definitions/DeliveryRuleCondition"
           }
         },
-        "conditions": {
-          "description": "A list of conditions that must be matched for the actions to be executed",
+        "actions": {
+          "description": "A list of actions that are executed when all the conditions of a rule are satisfied.",
```

Comment:
Any specific reason why the order was changed (conditions before actions)

Reply:
Max asked me to reorder it.